src/components/frame/v5/pages/DashboardPage/partials/MenuContentItem/MenuContentItem.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+import Icon from '~shared/Icon';
+import { formatText } from '~utils/intl';
+import { MenuContentItemProps } from './types';
+
+const displayName = 'v5.pages.DashboardPage.partials.Menu';
```

Comment:
displayName is incorrect here,also this component should be redundant if you will add a component with groups of links/buttons